frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/basicTicketImage.component.tsx

```diff
@@ -15,11 +15,13 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { ProjectsHooksSelectors } from '@/v5/services/selectorsHooks/projectsSelectors.hooks';
+import { FormControl, FormHelperText } from '@mui/material';
 import { ActionsList, ActionsSide, Container, PropertyName } from './basicTicketImage.styles';
 import { TicketImageDisplayer } from './ticketImageDisplayer/ticketImageDisplayer.component';
 
 export type BasicTicketImageProps = {
 	imgSrc: string,
+	formError: any,
```

Comment:
would prefer if instead of formError, would have error and helperText to mimic materialui's interface. 

That way in the near future we can get rid of all the Form... files and just have one that would apply a ReactHookform adapter what knows how to pass from react-hook-form to materialuis inputs..